### ISSUE TITLE
sendBetterEmail - File Name on Attachment is Incorrect

### ISSUE THREAD

**Issue by peanutman21** [alexed1/LightningFlowComponents#598]
I am using version 1.33.01 of sendBetterEmail

When sending out an email and there is an attachment, I am using the 'contentDocumentAttachments' input and including a collection of Content Document Links.
However, when I receive the email, instead of the name of the PDF showing the name of the File
(i.e. ABC Company, Inc. - 123456), it is showing the path from where the File was uploaded from
(i.e. CuserseschubertDesktopABC Company, Inc. - 123456.pdf)

### Steps to reproduce

1. Using Data Loader, insert a ContentVersion record of a PDF.
2. Based on the ContentVersion record created, create a ContentDocumentLink record to link the File just uploaded to an Account record.
3. Create a Flow. Create a collection which has the ContentDocumentLink record in it.
4. Use the sendBetterEmail Action and send the email.
5. When the email is received, the PDF File will be attached to email. However, the name of the File will be the path from the where the File was uploaded from, not the actual name of the File.
![Incorrect File Name](https://user-images.githubusercontent.com/78981726/107819334-9cbd8c80-6d3e-11eb-98a4-24dd6729853f.jpg)

Note: If you simply pull up an Account record and upload a File to the Account via the UI, and then repeat steps 3-5 above, the name of the File looks OK! This only seems to happen with Files uploaded via Data Loader.

### Expected behaviour
What I was expecting was the name of the PDF to be equal to the name of the file (Title), not the path of where the file was uploaded from.
![Correct File Name](https://user-images.githubusercontent.com/78981726/107820169-f70b1d00-6d3f-11eb-9a53-ec1cedd2386b.jpg)

### Actual behaviour
The name of the File is the path from the where the File was uploaded from not the actual name of the File.

### Debug Logs
Attached is the Debug Log if it helps. The Flow and the Action run fine. 
[Debug Log SendBetterEmail.log](https://github.com/alexed1/LightningFlowComponents/files/5974201/Debug.Log.SendBetterEmail.log)

**Comment by jdpond** [comment]
@peanutman21 Sorry, but I'm going to push this to the next release - my apologies, the Spring '21 hit me this weekend (completely my bad, Community Permissions).   Will try to move it in as soon as possible.

**Comment by peanutman21** [comment]
@jdpond - Not that I am going to hold you to it, but is there a date for when the next release is? Also, were you able to pinpoint the issue? I have a case open with Salesforce Support to try and help troubleshoot the issue as well. I'm happy to keep pursuing it with them if that will help you out. Let me know. Thanks!

**Comment by peanutman21** [comment]
@jdpond  - In my scenario, I uploaded Files with the Data Loader (created ContentVersion records) and then created ContentDocumentLink records to link the Files to the Account records. 

If I understood Salesforce Support correctly, in the scenario above, the Apex Class is setting the name of the file on the attachment of the outgoing email on line 204 of the Apex Class (Name = cv.PathOnClient,). If the Apex Class could use Title instead of PathOnClient, that would be best. 

(When I simply uploaded a File on the Account record via the UI, I guess the Apex Class never gets down to line 204 for the file name. I _think_ it gets set on line 204.) 
Hope this helps!
